pandas/core/dtypes/dtypes.py

```diff
@@ -886,7 +887,7 @@ def __new__(cls, freq):
             return u
 
     def __reduce__(self):
-        return type(self), (self.freq,)
+        return type(self), (self.name,)
```

Comment:
Will this break pickle compatibility?

Reply:
no. when it loads this gets passed to the constructor which converts the string